In-house patch to remove unnecessary dependencies from Heat's
requirements files. The specific reasons are as follows:

httplib2	Not applicable

kombu		Not applicable

pycrypto	Not applicable to Solaris (M2Crypto used instead)

qpid-python	Not applicable

cinderclient	push to version 1.3.1

--- heat-2015.1.2/heat.egg-info/requires.txt.orig	2015-10-13 10:55:15.000000000 -0600
+++ heat-2015.1.2/heat.egg-info/requires.txt	2015-11-19 15:08:10.240419209 -0700
@@ -2,10 +2,8 @@ pbr!=0.7,<1.0,>=0.6
 Babel>=1.3
 eventlet!=0.17.0,>=0.16.1
 greenlet>=0.3.2
-httplib2>=0.7.5
 iso8601>=0.1.9
 keystonemiddleware<1.6.0,>=1.5.0
-kombu>=2.5.0
 lxml>=2.3
 netaddr>=0.7.12
 oslo.config<1.10.0,>=1.9.3 # Apache-2.0
@@ -22,9 +20,8 @@ osprofiler>=0.3.0 # Apache-2.0
 oslo.versionedobjects<0.2.0,>=0.1.1
 PasteDeploy>=1.5.0
 posix-ipc
-pycrypto>=2.6
 python-ceilometerclient<1.2,>=1.1.1
-python-cinderclient<1.2.0,>=1.1.0
+python-cinderclient<=1.3.1,>=1.1.0
 python-glanceclient<0.18.0,>=0.15.0
 python-heatclient<0.5.0,>=0.3.0
 python-keystoneclient<1.4.0,>=1.2.0
@@ -34,7 +31,6 @@ python-saharaclient<0.9.0,>=0.8.0
 python-swiftclient<2.5.0,>=2.2.0
 python-troveclient<1.1.0,>=1.0.7
 PyYAML>=3.1.0
-qpid-python
 requests!=2.4.0,>=2.2.0
 Routes!=2.0,>=1.12.3
 six>=1.9.0

--- heat-2015.1.2/requirements.txt.orig	2015-10-13 10:51:54.000000000 -0600
+++ heat-2015.1.2/requirements.txt	2015-11-19 15:07:10.611579676 -0700
@@ -6,10 +6,8 @@ pbr!=0.7,<1.0,>=0.6
 Babel>=1.3
 eventlet!=0.17.0,>=0.16.1
 greenlet>=0.3.2
-httplib2>=0.7.5
 iso8601>=0.1.9
 keystonemiddleware<1.6.0,>=1.5.0
-kombu>=2.5.0
 lxml>=2.3
 netaddr>=0.7.12
 oslo.config<1.10.0,>=1.9.3 # Apache-2.0
@@ -26,9 +24,8 @@ osprofiler>=0.3.0 # Apache-2.0
 oslo.versionedobjects<0.2.0,>=0.1.1
 PasteDeploy>=1.5.0
 posix-ipc
-pycrypto>=2.6
 python-ceilometerclient<1.2,>=1.1.1
-python-cinderclient<1.2.0,>=1.1.0
+python-cinderclient<=1.3.1,>=1.1.0
 python-glanceclient<0.18.0,>=0.15.0
 python-heatclient<0.5.0,>=0.3.0
 python-keystoneclient<1.4.0,>=1.2.0
@@ -38,7 +35,6 @@ python-saharaclient<0.9.0,>=0.8.0
 python-swiftclient<2.5.0,>=2.2.0
 python-troveclient<1.1.0,>=1.0.7
 PyYAML>=3.1.0
-qpid-python
 requests!=2.4.0,>=2.2.0
 Routes!=2.0,>=1.12.3
 six>=1.9.0
